lib/http3/server.c

```diff
@@ -1702,9 +1702,13 @@ static void on_h3_destroy(h2o_quic_conn_t *h3_)
 {
     h2o_http3_conn_t *h3 = (h2o_http3_conn_t *)h3_;
     struct st_h2o_http3_server_conn_t *conn = H2O_STRUCT_FROM_MEMBER(struct st_h2o_http3_server_conn_t, h3, h3);
-
+    quicly_stats_t stats = {0};
```

Comment:
```suggestion
    quicly_stats_t stats;
```
Generally speaking, I do not think we initialize "out" parameters. Leaving them as they are gives the static analyzers better chance of detecting errors.

Reply:
OK, I will remove it in the next update.